### PR TITLE
Subversion

### PR DESCRIPTION
**Pull request by scopatz**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by scopatz** [comment]
@conda-forge/staged-recipes - this is ready for review!

File: recipes/subversion/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    - lz4-c
+    - libutf8proc
+    - gettext
+    - expat
+    - gnupg  # [unix]
+    - cyrus-sasl  # [unix]
+    - libmagic  # [unix]
+    - serf  # [linux]
+  run:
+    - libapr
```
Comment (xhochy) [inline recipes/subversion/meta.yaml:43]: Are these libraries missing a run_exports?

Comment (scopatz) [reply recipes/subversion/meta.yaml:43]: Probably - that would explain why I need to put them in run deps

**Comment by scopatz** [comment]
ping @conda-forge/staged-recipes

**Comment by jakirkham** [comment]
Thanks @scopatz! 😄

File: recipes/subversion/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipes/subversion/meta.yaml:43]: Both [`libapr`]( https://github.com/conda-forge/apr-feedstock/blob/f2b2554770474d280b0c67d26eefc2f75f6f04d2/recipe/meta.yaml#L44-L47 ) and [`libaprutil `]( https://github.com/conda-forge/apr-feedstock/blob/f2b2554770474d280b0c67d26eefc2f75f6f04d2/recipe/meta.yaml#L71-L72 ) have `run_exports`.

File: recipes/subversion/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+{% set name = "subversion" %}
```
Comment (jakirkham) [inline recipes/subversion/meta.yaml:1]: In `defaults` this is named [`svn`]( https://anaconda.org/anaconda/svn ).

Comment (scopatz) [reply recipes/subversion/meta.yaml:1]: In defaults, `svn` in the Python subversion wrapper, which is why I called 
 this `subversion` https://anaconda.org/anaconda/svn

**Pull request merged**